metashare/api/serializers.py

```diff
@@ -188,6 +181,8 @@ class Meta:
             "pr_url",
             "status",
             "pr_is_open",
+            "assigned_dev",
+            "assigned_qa",
```

Comment:
@wlonk I think we maybe need a validator for these, making sure the user is already in `project.github_users`? Tho that might depend on the answer to my first question here: https://trello.com/c/TbBcu3Zu/22-i-want-to-assign-team-members-to-a-specific-task#comment-5e2e156a2623a30e7fdad423

That is, if a user is assigned to a Task, _then_ removed from the Project -- I'm not sure if we want subsequent updates to the Task to fail validation or not...

Reply:
Right. I think we _don't_ but I wanna talk this over with @davisagli at least.

Reply:
Ok... in that case we could validate only when a user is added, or not at all on the back end, and allow the front end to limit the available options.